lib/ssl/test/tls_server_session_ticket_SUITE.erl

```diff
@@ -114,19 +126,25 @@ end_per_testcase(_TestCase, Config) ->
 main_test() ->
     [{doc, "Ticket main scenario"}].
 main_test(Config) when is_list(Config) ->
+    case ?config(anti_replay, Config) of
+        undefined -> undefined;
+        _ ->
+            % Sleep longer than warmup period
+            ct:sleep({seconds, 2 * ?LIFETIME})
+    end,
```

Comment:
3 occurences of ct:sleep case statements for anti_replay tests - could you move them to init_per_testcase.
the less sleeping and copy/code code the better.

Reply:
I moved it into `init_per_testcase`, the difference is that now we also sleep for `misc_test` (4 extra seconds). I don't see a reason to make an auxiliary function taking into account which test to sleep in, unless more anti-replay tests are added that need to not sleep.

Reply:
Ah. I missed misc_test that one. I agree it shouldn't be a problem. Actually I think this should be a default behavior for anti_replay tests since PR for customizing seed.